islands/notifications.tsx

```diff
@@ -61,40 +107,73 @@ export const NotificationDetail = (success: SuccessType) => {
     setErrors(errors || {});
 
     if (errors) {
+      console.error("submit errors", errors);
       e.preventDefault();
       return;
     }
   };
+
+  useEffect(() => {
+    if (data?.type === "2") {
```

Comment:
Better to reference the type by name than number for legibility, `NotificationType[data?.type]`